tests/testapp/tests/test_morango_mptt.py

```diff
@@ -0,0 +1,55 @@
+from django.test import TestCase
+from facility_profile.models import Facility
+from facility_profile.models import MyUser
+from morango.utils.morango_mptt import MorangoTreeQuerySet, MorangoMPTTTreeManager
+
+
+class MorangoMPTTModelTestCase(TestCase):
+
+    def setUp(self):
+        Facility.objects.create(name='beans')
+
+    def test_mptt_manager_inheritance(self):
+        self.assertTrue(isinstance(Facility.objects, MorangoMPTTTreeManager))
+        self.assertFalse(isinstance(MyUser.objects, MorangoMPTTTreeManager))
+
+    def test_mptt_qs_inheritance(self):
+        self.assertTrue(isinstance(Facility.objects.all(), MorangoTreeQuerySet))
+        self.assertFalse(isinstance(MyUser.objects.all(), MorangoTreeQuerySet))
+
+    def test_mptt_manager_update(self):
+        self.assertTrue(Facility.objects.first()._dirty_bit)
+        Facility.objects.update(update_dirty_bit_to=False)
+        self.assertFalse(Facility.objects.first()._dirty_bit)
+        Facility.objects.update(update_dirty_bit_to=None)
+        self.assertFalse(Facility.objects.first()._dirty_bit)
+        Facility.objects.update(update_dirty_bit_to=True)
+        self.assertTrue(Facility.objects.first()._dirty_bit)
+
+    def test_mptt_qs_update(self):
+        self.assertTrue(Facility.objects.first()._dirty_bit)
+        Facility.objects.all().update(update_dirty_bit_to=False)
+        self.assertFalse(Facility.objects.first()._dirty_bit)
+        Facility.objects.all().update(update_dirty_bit_to=None)
+        self.assertFalse(Facility.objects.first()._dirty_bit)
+        Facility.objects.all().update(update_dirty_bit_to=True)
+        self.assertTrue(Facility.objects.first()._dirty_bit)
+
+    def test_mptt_save(self):
+        fac = Facility.objects.first()
+        self.assertTrue(fac._dirty_bit)
+        fac.save(update_dirty_bit_to=False)
+        self.assertFalse(Facility.objects.first()._dirty_bit)
+        fac.save(update_dirty_bit_to=None)
+        self.assertFalse(Facility.objects.first()._dirty_bit)
+        fac.save(update_dirty_bit_to=True)
+        self.assertTrue(Facility.objects.first()._dirty_bit)
```

Comment:
Maybe this is handled elsewhere, but making sure that a plain save (with no `update_dirty_bit_to` argument provided) also sets the dirty bit would be good.